src/LibraryManager.Vsix/UI/Controls/Library.xaml.cs

```diff
@@ -257,5 +266,10 @@ private void ThisControl_GotFocus(object sender, RoutedEventArgs e)
                 LibrarySearchBox.Focus();
             }
         }
+
+        private void FilterOutUnmatchedItems(ref IEnumerable<CompletionItem>items, string versionPrefix)
```

Comment:
Nit: versionSuffix.

Also, why not just return the filtered enum instead of passing in original by ref?